### PR TITLE
Added remove_texts method in WeaviateSearch

### LINKED ISSUE #512: Add `remove_texts` methods to all `Langchain::Vectorsearch::*` classes
Similar to https://github.com/andreibondarev/langchainrb/pull/510, we'd like to add this method to all other vectorsearch classes. Please split these out into separate PRs.

- [x] Chroma#remove_texts
- [ ] Elasticsearch#remove_texts
- [ ] Epsilla#remove_texts
- [ ] Hnwslib#remove_texts
- [ ] Milvus#remove_texts
- [x] Pgvector#remove_texts
- [x] Pinecone#remove_texts
- [x] Qdrant#remove_texts
- [x] Weaviate#remove_texts

### PR DESCRIPTION
**Pull request by capak07**
- Fixes #512 
- Added remove_texts method in Weaviate

### CODE REVIEW

File: lib/langchain/vectorsearch/weaviate.rb [hunk 1]
```diff
@@ -71,6 +71,15 @@ def update_texts(texts:, ids:)
       end
     end
 
+    def remove_texts(ids:)
```
Comment (andreibondarev) [inline lib/langchain/vectorsearch/weaviate.rb:74]: Thoughts on using the [batch_delete](https://github.com/andreibondarev/weaviate-ruby/blob/main/lib/weaviate/objects.rb#L160) method?

Comment (capak07) [reply lib/langchain/vectorsearch/weaviate.rb:74]: Sorry didn't see this, wanted to check on what would be the right path for the `where `param, I had something like this
`{ path: [\"__id\"], operator: Equal, valueString: \"#{ids}\" `?

Comment (andreibondarev) [reply lib/langchain/vectorsearch/weaviate.rb:74]: Batch delete docs: https://weaviate.io/developers/weaviate/api/rest/batch#batch-delete

But take a look at the where filter specs: https://weaviate.io/developers/weaviate/api/graphql/filters#where-filter

You may need to use `"ContainsAny":
image

**Comment by andreibondarev** [comment]
@capak07 I couldn't push to your branch so I fixed and pushed to origin: https://github.com/andreibondarev/langchainrb/pull/568. It's merged now! Thank you!

**Comment by capak07** [comment]
> @capak07 I couldn't push to your branch so I fixed and pushed to origin: #568. It's merged now! Thank you!

No worries, thank you.
I hope I can contribute more to this repository

**Pull request merged**